src/sentry/static/sentry/app/views/performance/keyTransactionQuery.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+
+import withApi from 'app/utils/withApi';
+import {Client} from 'app/api';
+import {Organization} from 'app/types';
+import {saveKeyTransaction, deleteKeyTransaction} from 'app/actionCreators/performance';
+
+type ChildrenProps = {
+  isLoading: boolean;
+  error: string | null;
+  isKeyTransaction: boolean;
+  toggleKeyTransaction: () => void;
+};
+
+type Props = {
+  api: Client;
+  projectID: number;
+  organization: Organization;
+  transactionName: string;
+  children: (props: ChildrenProps) => React.ReactNode;
+};
+
+type State = {
+  isLoading: boolean;
+  keyFetchID: symbol | undefined;
+  error: null | string;
```

Comment:
Seems to me that using `null` makes is more explicit here as there can be other reasons why a variable is `undefined` (e.g. simply forgetting to initialize it or misspelling the key) but the only way it can be `null` is if it was explicitly set to null.

Reply:
Should keyFetchID be `symbol | null` then?

Reply:
Good point, I'll fix that.